models/staging/_staging.yml

```diff
@@ -1,15 +1,618 @@
 version: 2
 
 models:
+    
```

Comment:
At some point, these definitions should live in `models/analytics` core models. The _staging.yml file will only be used to define primary key tests for each model. Otherwise it'd get pretty burdensome to document the same columns across multiple parts of the project.